internal/artifacts/testdata/mtahtml5/mtaWithBuildParams.yaml

```diff
@@ -13,30 +13,18 @@ modules:
     - name: uaa_mtahtml5
     - name: dest_mtahtml5
    build-parameters:
+    builder: zip
     requires:
     - name: ui5app2
 
-
  - name: ui5app2
    type: html5
    path: ui5app2
-   parameters:
-      disk-quota: 256M
-      memory: 256M
-   requires:
-   - name: uaa_mtahtml5
-   - name: dest_mtahtml5
-
-resources:
- - name: uaa_mtahtml5
-   parameters:
-      path: ./xs-security.json
-      service-plan: application
-   type: com.company.xs.uaa
-
- - name: dest_mtahtml5
-   parameters:
-      service-plan: lite
-      service: destination
-   type: org.cloudfoundry.managed-service
+   build-parameters:
+      builder: zip
 
+ - name: ui5app3
+   type: html5
+   path: ui5app2
```

Comment:
Should the path here be `ui5app3`?

Reply:
i didn't want to create more artifacts, but yes, it's not good, I'm fixing